mailpoet/lib/Models/MappingToExternalEntities.php

```diff
@@ -2,12 +2,37 @@
 
 namespace MailPoet\Models;
 
+/**
+ * @deprecated This model is deprecated and there is no replacement.
+ * This class can be removed after 2023-01-06.
+ */
```

Comment:
I think that because the table is not present anymore, the usage of this model will, apart from the deprecation warnings, cause also PHP Errors when trying to save stuff to DB, etc. Maybe, in this case, we could make an exception and remove the model right away. What do you think?

Reply:
Please correct if I'm wrong, but I believe the table will still exist for sites that activated the plugin before this change. In this PR, we are stopping the creation of the table for new installs, but we are not dropping the table for sites that already have it. So for the odd site that still has this table and for some reason is using this model directly, I would prefer to deprecate the model instead of removing it. I think there is a chance no such site exists, but anyway, I would suggest we play it safe and follow the deprecation path instead of removing the model straight away.

Reply:
> Please correct if I'm wrong, but I believe the table will still exist for sites that activated the plugin before this change. 
Good point. Let's keep the model.